packages/web/src/index.tsx

```diff
@@ -13,9 +14,35 @@ import FmsFileExplorer from "../../core/App";
 import { createReduxStore } from "../../core/state";
 import FileViewerServiceWeb from "./services/FileViewerServiceWeb";
 import FileDownloadServiceWeb from "./services/FileDownloadServiceWeb";
+import Layout from "./components/Layout";
+import Home from "./components/Home";
 
 const APP_ID = "fms-file-explorer-web";
 
+const router = createBrowserRouter([
+    {
+        element: <Layout />,
+        children: [
+            {
+                path: "/",
+                element: <Home />, // Splash page
+            },
+            {
+                path: "app",
+                element: <FmsFileExplorer />,
+            },
+            {
+                path: "datasets",
+                element: <></>, // Datasets placeholder
+            },
+            {
+                path: "learn",
```

Comment:
For the initial release lets exclude "Learn" as an option. We won't have enough concrete examples to put in there in time

Reply:
Removed in [16ec93a](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/107/commits/16ec93a7118115877bbf37cba30987c8fd5483d9)